master/usage/configuration/bgp.md

```diff
@@ -96,21 +96,54 @@ to ensure continuity of service.
 
 #### Example
 
-To turn off the full BGP node-to-node mesh, or to modify the global AS number,
```

Comment:
shouldn't we keep these two lines? Example kinda just starts with the bullet points without these two lines explaining what's going on

Reply:
It's a good point.